content/docs/getting-started.md

```diff
@@ -63,13 +63,11 @@ It can be run standalone, but requires Jaeger backend to view the
 #### Running
 
 ```bash
-mkdir -p $GOPATH/src/github.com/jaegertracing
-cd $GOPATH/src/github.com/jaegertracing
-git clone git@github.com:jaegertracing/jaeger.git jaeger
-cd jaeger
-make install
-cd examples/hotrod
-go run ./main.go all
+$ docker run --rm -it \
```

Comment:
I think it's good to add a docker command as an easier way to run, but we should keep the original instructions as well, because the point of this application, as described in the blog post, is to find and solve performance issues, which requires running from source.

Reply:
I will include both the steps to run Hotrod example